src/50_Sample_AMPs/Recipe.html

```diff
@@ -0,0 +1,460 @@
+      Published: June 12, 2016</small></p>
+  </div>
+
+  <!-- #### Social Sharing  -->
+
+  <!--
+  The Social Share extension provides a common interface for share buttons.  Learn more about [`amp-social-share`](/components/amp-social-share/).
+  -->
+  <p class="heading">
+    <amp-social-share type="twitter" data-param-text="Amazing recipe that my friend made" width="45" height="33"></amp-social-share>
+    <amp-social-share type="facebook" width="45" height="33" data-attribution=254325784911610></amp-social-share>
+    <amp-social-share type="gplus" width="45" height="33"></amp-social-share>
+    <amp-social-share type="email" width="45" height="33"></amp-social-share>
+    <amp-social-share type="pinterest" width="45" height="33"></amp-social-share>
+  </p>
+
+
+  <figure>
+    <amp-img src="/img/lamb-garlic-done.jpg" width="1280" height="479" layout="responsive" attribution="Dan Dascalescu"></amp-img>
+    <figcaption>Garlic-studded roast lamb. Serve with a side of baked potatoes.</figcaption>
+  </figure>
+
+  <p>This is a traditional Romanian lamb recipe, with the subtle taste of garlic deeply permeating the meat. It is customarily prepared for Easter, but can be enjoyed year-round. For best results, stud the meat a day in advance.</p>
+  <!-- #### Image Galleries  -->
+  <!--
+  Use `amp-carousel` for image galleries. Here we display responsively images with different aspect ratios. Learn more about [creating image galleries in AMP](/advanced/image_galleries_with_amp-carousel/).
+  -->
+  <amp-carousel width="1280" height="970" layout="responsive" type="slides">
+    <figure>
+      <amp-img src="/img/meat-silverskin-cutting.jpg" width="401" height="542" layout="fill" attribution="Scott Phillips"></amp-img>
+      <figcaption>Prepare the meat by washing and removing the silverskin.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/garlic.jpg" width="1280" height="960" layout="fill" attribution="Wikipedia"></amp-img>
+      <figcaption>Prepare the garlic.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/lamb-with-garlic-slivers.jpg" width="1021" height="763" layout="fill" attribution="Kara / EatDrinkAdventure"></amp-img>
+      <figcaption>Insert slivers of garlic into the meat; the deeper the better.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/tomatoes-cut.jpg" width="720" height="720" attribution="CC0 - https://pixabay.com/en/chopped-tomatoes-tomato-food-red-1375892/" layout="fill"></amp-img>
+      <figcaption>Cut the tomatoes.</figcaption>
+    </figure>
+    <figure>
+      <amp-img src="/img/lamb-garlic-ready-for-oven.jpg" width="1280" height="1703" attribution="Dan Dascalescu"  layout="fill"></amp-img>
+      <figcaption>The garlic lamb roast is ready for baking.</figcaption>
+    </figure>
+  </amp-carousel>
+
+
+  <!-- We should switch to the [AMP star rating component](https://github.com/ampproject/amphtml/issues/2691) when it becomes available. -->
```

Comment:
s/We should switch to the [AMP star rating component](https://github.com/ampproject/amphtml/issues/2691) when it becomes available./An [AMP star rating component](https://github.com/ampproject/amphtml/issues/2691) is currently being developed.